util/pace/util/grid/eta.py

```diff
@@ -204,14 +204,372 @@ def set_hybrid_pressure_coefficients(km: int) -> HybridPressureCoefficients:
                 1,
             ]
         )
+
+    elif km == 91:
```

Comment:
I think this is fine since this is how we set up our npz=79 case, but we should probably merge [this](https://github.com/ai2cm/pace/blob/f2aa9f4d827a63907cff5b4253e626f07f21b612/driver/pace/driver/grid.py#L231) so it is possible to replace ak/bk with custom values.